pathml/datasets/pannuke.py

```diff
@@ -363,7 +367,7 @@ def train_dataloader(self):
         Yields (image, mask, tissue_type), or (image, mask, hv, tissue_type) for HoVer-Net
         """
         return data.DataLoader(
-            dataset=self._get_dataset(fold_ix=self.split),
+            dataset=self._get_dataset(fold_ix=self.split, augment=True),
```

Comment:
Code looks solid. I did however find it difficult to understand the code for how the dataset is split. 
```python
if self.split in [1, 2]:
            fold_ix = 3
        else:
            fold_ix = 1
```
I checked in `PanNukeDataset` as well and couldnt find much. I think it would be worth it to clarify the difference in folding between indices 1, 2 or 3.